src/tab/_tab.scss

```diff
@@ -0,0 +1,12 @@
+.tab {
+  .tab-header {
+    display: flex;
+  }
+
+  .tab-header-item {
+    --button-bg: transparent;
+    --button-width: auto;
+
+    color: black;
+  }
```

Comment:
Let's not nest the selectors if it isn't necessary. This will make overriding easier